freshdesk_api.py

```diff
@@ -186,7 +187,11 @@ def req(self, func, path, resource_type, params={}, **kwargs):
                 # Either we get a list or a single object
                 json_obj = resp.json()
                 if isinstance(json_obj, (list, tuple)):
-                    return [i[resource_type] for i in json_obj]
```

Comment:
Would better see this implemented as :

- if `resource_type` is none, do not unpack
- in `Ticket.get()` call `self.api.get(..., resource_type=None)`